docs/playbook/ingesting-data/005-ingesting-dynamo-db-tables.md

```diff
@@ -0,0 +1,178 @@
+1. Navigate to [SSM parameter store][aws_console_ssm] in the AWS console.
+1. Click "Create parameter".
+1. Set the name of the parmeter to be something like `/where-the-data-is-coming-from/prod/role-arn-to-access-dynamodb-tables`.
+1. Keep note of the parameter name, as you will need it in the next section, and click "Create parameter".
+
+## Setting up the AWS Glue job
+
+1. Open the [Data Platform Project](https://github.com/LBHackney-IT/data-platform). If you don't have the correct permissions, you'll get a '404' error (see [prerequisites](#prerequisites)).
+2. Navigate to the main `terraform` directory and add a new file named after the dataset you are ingesting, for example `30-mtfh-tables-ingestion.tf`. Click on "Add file" then "Create new file" to do this.
+3. Load the role ARN saved in the [previous section](#saving-the-role-arn-in-the-data-platform-accounts) in to terraform.
+  - Copy this example code block into the file.
+  - Rename `role_arn_to_access_housing_tables` to be relevent to the kind of tables you are ingesting.
+  - Change the name `/where-the-data-is-coming-from/prod/role-arn-to-access-dynamodb-table` to be the name of the parameter you created in the previous section.
+
+  ```tf
+  data "aws_ssm_parameter" "role_arn_to_access_housing_tables" {
+    name = "/where-the-data-is-coming-from/prod/role-arn-to-access-dynamodb-table"
+  }
+  ```
+
+4. Create the glue job & crawler.
+Copy the example terraform module below into the file.
+This is example terraform to create a glue job in the Data Platform account.
+You will then need to changes the following values
+   - The module name - Declared at the top, in the example it is set to "ingest_dataset_tables. You should rename this to reflect the entire collection of data that is being ingested this job.
+   - `job_name` - Name of your job with `${local.short_identifier_prefix}` prefixed in front of it like the example.
+   - `job_description` (Optional) - An optional description for what this job is doing.
+   - `database_name` - The name of the glue catalogue database where the tables will be writted to in the data platform. 
+   If you want to create the tables in the landing zone database, this will be `aws_glue_catalog_database.landing_zone_catalog_database.name`.
+   If you want to create the tables in a departments raw zone database, this will be `module.department_DEPARTMENT_NAME.raw_zone_catalog_database_name`.
+   - `--table_names` - A comma delimited list of table names to ingest from Dynamo DB.
+   - `--role_arn` - This should be the value of the data object created in the step above, for the example given it would be `data.aws_ssm_parameter.role_arn_access_housing_tables.value`. 
+   This signifies to terraform to go and get the value of this parameter, in which you have saved the ARN of the role created in the initial step. 
+   So eventually this will result to that role ARN, but it allows you tos configure a different role ARN for each environment.
```

Comment:
`result to that role ARN` reads incorrectly to me, I'm not sure what's being said
`allows you tos configure ` -> `allows you tos configure`